tests/__tests__/parser/validate.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+require('babel-polyfill');
+const validateAddons = require('../../../lib/utils/validate-addons');
+
+describe('Addon Validation', () => {
+	it('Should notify if the package isn\'t prefixed', async () => {
+		try {
+			await validateAddons(['webpack']);
+		} catch (object) {
+			expect(object.toString()).toEqual('TypeError: Package isn\'t a valid name\n' +
+			'It should be prefixed with \'webpack-addon\'');
+		}
+	});
+	it('Should complain about only supplying \'webpack-addon\' as the name', async () => {
+		try {
+			await validateAddons(['webpack-addon']);
+		} catch (object) {
+			expect(object.toString()).toEqual('TypeError: \'webpack-addon\' is not a valid addon');
+		}
+	});
+	it('Should complain about unregistered packages', () => {
+		// error shows up in console as we're handling it instead of throwing an actual error.
+		expect(validateAddons(['webpack-addon-trinity'])).toBeDefined();
```

Comment:
I'm wondering what this does. Anyways I'm having issues with Jest quitting on every run because of https://github.com/webpack/webpack-cli/pull/27/files#diff-3b9090788549586f699038f130f8a542R16 and the fact it isn't wrapped.

<img width="477" alt="2017-02-03 at 17 41" src="https://cloud.githubusercontent.com/assets/11071/22599716/121169e0-ea38-11e6-8c2f-00af890376c4.png">